.github/workflows/ci.yaml

```
@@ -17,20 +17,28 @@ jobs:
         uses: actions/setup-go@v5
         with:
           go-version: ${{ matrix.go-version }}
+      - name: Test
+        run: make test
+
+  lint:
+    name: Linting
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install Go ${{ matrix.go-version }}
```

```suggestion
      - name: Install Go
```

`matrix.go-version` is undefined now as it was removed from the matrix definition